From e2fd12e93e078ba8acd1b0cc256e4c244bd9b0cf Mon Sep 17 00:00:00 2001
From: olegsvs <oleg.texet@gmail.com>
Date: Sun, 15 May 2016 14:01:46 +0300
Subject: [PATCH 1134/1168] Revert "write cfg to touchscreen"

This reverts commit 6fc0f08e650a996ea7a85d1aff08562cc60d6e70.

Conflicts:
	drivers/input/touchscreen/mediatek/GT915_HD/tpd_custom_gt9xx.h
---
 drivers/input/touchscreen/mediatek/GT915_HD/tpd_custom_gt9xx.h | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/drivers/input/touchscreen/mediatek/GT915_HD/tpd_custom_gt9xx.h b/drivers/input/touchscreen/mediatek/GT915_HD/tpd_custom_gt9xx.h
index 53ab998..8c91e07 100755
--- a/drivers/input/touchscreen/mediatek/GT915_HD/tpd_custom_gt9xx.h
+++ b/drivers/input/touchscreen/mediatek/GT915_HD/tpd_custom_gt9xx.h
@@ -135,10 +135,17 @@ extern s32 gtp_i2c_read_dbl_check(struct i2c_client *client, u16 addr, u8 *rxbuf
 
 #define GTP_GESTURE_WAKEUP    0 
 
+<<<<<<< HEAD
 #define TPD_PROXIMITY
 //#define TPD_HAVE_BUTTON             // report key as coordinate,Vibration feedback
 #define TPD_WARP_X                  // mirrored x coordinate
 #define TPD_WARP_Y                  // mirrored y coordinate
+=======
+//#define TPD_PROXIMITY
+#define TPD_HAVE_BUTTON            // report key as coordinate,Vibration feedback
+//#define TPD_WARP_X                  // mirrored x coordinate
+//#define TPD_WARP_Y                  // mirrored y coordinate
+>>>>>>> parent of 6fc0f08... write cfg to touchscreen
 #define GTP_DEBUG_ON          1
 #define GTP_DEBUG_ARRAY_ON    0
 #define GTP_DEBUG_FUNC_ON     0
-- 
2.7.4

